.pullapprove.yml

```diff
@@ -0,0 +1,45 @@
+version: 2
+
+group_defaults:
+  always_pending:
+    labels:
+      - do-not-merge
+      - needs-rebase
+      - work-in-progress
+  approve_by_comment:
+    enabled: true
+    approve_regex: '^:+1:'
+  author_approval:
+    ignored: true
+
+groups:
+  develop-reviewers:
+    required: 1
```

Comment:
We've only been using 1 for Clio. We might want to switch to 2 for develop and 1 for staging, since reviews for merges to staging are probably going to be rubber-stamps on a huge wall of changes (case in point).